live-backend/.env.example

```
@@ -0,0 +1,8 @@
+#Postgres DB url with timescale extension activated on it
+DATABASE_URL=
+#Websocket server url
+WS_URL=
+#Origin for CORS
+ORIGIN=
+#Log level
+LOG_LEVEL=info
```

The logging has changed since and I didn't update example.env so this is the new way:
```suggestion
RUST_LOG="live_backend=debug,info"
```